packages/perseus/src/perseus-types.ts

```diff
@@ -3,6 +3,9 @@
 
 import type {Coord} from "./interactive2/types";
 
+export type Range = [min: number, max: number];
+export type Size = [width: number, height: number];
```

Comment:
I'm not familar with this, but I'm interpreting it as `[number, number]`, but you're naming the indices?

Reply:
It's a helper syntax to give more meaning to the tuple.  You're exactly correct, they're just labels/names, but I don't think you can even reference the tuple items using them, so they're strictly decorative. They are picked up by the editor which gives the hint as well. 

<img width="500" alt="image" src="https://user-images.githubusercontent.com/77138/234371574-4347ca4a-3964-47c3-9062-931de4af8242.png">